.github/workflows/build.yml

```diff
@@ -33,32 +41,36 @@ jobs:
             nx-${{ env.NX_BRANCH }}-
             nx-
 
-      - name: Install Dependencies
-        run: npm ci --prefer-offline --no-audit
+      - run: sudo apt-get update && sudo apt-get install -y build-essential libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev librsvg2-dev pkg-config
 
-      - name: Bootstrap packages
-        run: npm run bootstrap
+      - name: Install Dependencies
+        run: pnpm install --prefer-offline && pnpm -r rebuild
 
       - name: Lint packages
-        run: npm run lint
+        run: pnpm run lint
 
       - name: Build packages
-        run: npm run build
+        run: pnpm run build
 
       - name: Run unit tests
-        run: npm run test
+        run: pnpm run test
 
       - name: Upload webpack stats artifact (editor)
         uses: relative-ci/agent-upload-artifact-action@v2
         with:
           webpackStatsFile: ./playground/dist/webpack-stats.json
           artifactName: relative-ci-artifacts-editor
 
+      - name: Soft release
+        id: soft-release
+        run: pnpx pkg-pr-new publish './packages/*' # --compact enable compact after release
```

Comment:
This adds support for [pkg.pr.new](https://github.com/stackblitz-labs/pkg.pr.new) which allows us to publish preliminary versions of packages to their registry, which should decrease the burden of releasing things when people need them as they can always use the preliminary version. Also great for testing things out in development

Reply:
Can see an example on this PR: https://github.com/TypeCellOS/BlockNote/pull/1567#issuecomment-2761268580

Reply:
It allows you to install the "final" package into another repository so it is very useful for testing in a real project.
Like, for that bug fix for docs, we could have made a PR with the change and they could install this version instead of having to actually release something